java-lib/src/main/java/com/wavefront/api/agent/ValidationConfiguration.java

```diff
@@ -212,6 +218,14 @@ public ValidationConfiguration setLogAnnotationsCountLimit(int value) {
     return this;
   }
 
+  public boolean isEnableHyperlogsConvergedCsp() {
```

Comment:
nit: can we rename this to `hyperlogsConvergedCspEnabled`?

Reply:
or `enableHyperlogsConvergedCsp`

Reply:
done